src/lib/components/molecules/waffled-squares/index.jsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { SvgSquare } from '$particles/svg-square'
+
+export const WaffledSquare = ({
+  name = 'Germany',
+  squaresTotal = 96,
+  waffleWidth = 100,
+  waffleHeight = waffleWidth,
+  rowLong = 10,
+  squareWidth = waffleWidth / rowLong,
```

Comment:
This should probably be deduced from the width of the component at render time and the `rowLong` prop. See earlier comment about responsiveness.